readme.md

```diff
@@ -42,6 +42,10 @@ gulp.task('default', () =>
 
 #### options
 
+gulp-mocha will pass any options defined directly to the `mocha` binary. That
+means you have every [command line option](http://mochajs.org/#usage) available
```

Comment:
Needs to be explicit about it users using a camelCased version of those command line options.